sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -58,6 +58,8 @@ object ScalaJSPlugin extends AutoPlugin {
     val scalaJSIsSnapshotVersion = ScalaJSVersions.currentIsSnapshot
     val scalaJSBinaryVersion = ScalaJSCrossVersion.currentBinaryVersion
 
+    val ScalaJSTags = org.scalajs.sbtplugin.ScalaJSTags
```

Comment:
Maybe this could be an `object` inline, rather than duplicating its contents in a separate object? @gzm0 Opinion?

Reply:
Yeah, the only reason I used an object is because it was used in the task definition and I though it would be a bit ugly to use something from the ScalaJSPlugin in the ScalaJSPluginInternal 😃

Reply:
> it would be a bit ugly to use something from the ScalaJSPlugin in the ScalaJSPluginInternal

We do that all the time. Those two objects should be viewed as one, with the non-Internal one containing the public API, and Internal one everything else.